src/usr/local/www/services_unbound_advanced.php

```diff
@@ -341,6 +350,15 @@
 	$pconfig['dnsrecordcache']
 ))->setHelp('When enabled, allows unbound to serve one query even with a TTL of 0, if TTL is 0 then new record will be requested in the background when the cache is served to ensure cache is updated without latency on service of the DNS request.');
 
+$section->addInput(new Form_Checkbox(
+	'aggressivensec',
+	'Aggressive NSEC',
+	'Aggressive Use of DNSSEC-Validated Cache',
+	$pconfig['aggressivensec']
+))->setHelp('When enabled, unbound uses the DNSSEC NSEC chain to synthesize NXDOMAIN and other denials, ' .
+	    'using information from previous NXDOMAINs answers. It helps to reduce the query rate towards ' .
+	    'targets that get a very  high  nonexis‐tent name lookup rate.');
```

Comment:
A couple issues here, looks like from copy/pasting this description. There are extra spaces around `high`, and a hyphen in the middle of `nonexis-tent` which needs to be removed.

Reply:
oops! fixed